src/common/code_utils.hpp

```diff
@@ -89,14 +90,14 @@
  * @param[in] aMessage  A message (text string) to print on failure.
  *
  */
-#define SuccessOrDie(aStatus, aMessage)                                      \
-    do                                                                       \
-    {                                                                        \
-        if ((aStatus) != 0)                                                  \
-        {                                                                    \
-            otbrLogEmerg("FAILED %s:%d - %s", __FILE__, __LINE__, aMessage); \
-            exit(-1);                                                        \
-        }                                                                    \
+#define SuccessOrDie(aStatus, aMessage)                                                   \
+    do                                                                                    \
+    {                                                                                     \
+        if ((aStatus) != 0)                                                               \
+        {                                                                                 \
+            otbrLogEmerg("FAILED %s:%d - %s: %d", __FILE__, __LINE__, aMessage, aStatus); \
```

Comment:
How about putting the status value in front of the message string? The length of the message string is uncertain. Putting the status value at the beginning makes the developer easier to find the status value.

Reply:
SGTM, updated!